modules/timeline/src/main/EntryApi.scala

```diff
@@ -8,8 +8,10 @@ import scala.concurrent.duration._
 import lila.common.config.Max
 import lila.db.dsl._
 import lila.hub.actorApi.timeline.Atom
+import lila.hub.actorApi.timeline.BlogPost
 import lila.memo.CacheApi._
 import lila.user.User
+// import lila.blog.BlogApi doesnt work
```

Comment:
how can i make this work? error says blog is not recognized as a member of lila
and i meant to type Langs instead of BlogApi

Reply:
ok nvm found the fix